package.json

```diff
@@ -2,8 +2,8 @@
   "name": "tailor-teaching-elements",
   "version": "1.4.0-beta.6",
   "description": "Tailor teaching elements",
-  "main": "dist/tailor-teaching-elements.min.js",
-  "module": "dist/tailor-teaching-elements.es.js",
+  "main": "dist/tailor-teaching-elements.min.umd.js",
+  "module": "dist/tailor-teaching-elements.min.esm.js",
```

Comment:
Is there any gain in pointing to minimized esm? Esm is always bundled anyway?

Reply:
It was not minimized before so I tried to see if the minimized file is significantly smaller than the regular one so I replaced it. If you think it will cause issues or is unnecessary then I will remove it.

Reply:
Thing is esm can't be used as is. It has to be bundled unless you aren't using native esm modules support together with http2 which would be brave and not an easy ride atm. Bundling already minified code is less effective than doing it on original source due to lack of context.

Reply:
Good to know. It is removed now.